docs/diagnostics/PX1055.md

```diff
@@ -0,0 +1,68 @@
+# PX1055
+This document describes the PX1055 diagnostic.
+
+## Summary
+
+| Code   | Short Description                          | Type  | Code Fix  | 
+| ------ | ------------------------------------------ | ----- | --------- | 
+| PX1055 | An invalid primary key of the DAC is used. | Error | Available | 
+
+## Diagnostic Description
+The key fields of a DAC can be defined based on the following guidelines:
+
+ - A field bound to an identity column (that is, a field that has the `PXDBIdentity` or `PXDBLongIdentity` attribute) can be the only key field in the DAC.
+ - A DAC can have multiple key fields if none of these key fields is bound to an identity column.
```

Comment:
It is not clear that a DAC can have one non-identity key field (very common case).
The purpose of this diagnostic is to avoid mixing identity and non-identity fields in the primary key.